src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableList_1.cs

```diff
@@ -920,7 +921,10 @@ IImmutableList<T> IImmutableList<T>.InsertRange(int index, IEnumerable<T> items)
         /// <returns>
         /// A <see cref="IEnumerator{T}"/> that can be used to iterate through the collection.
         /// </returns>
-        IEnumerator<T> IEnumerable<T>.GetEnumerator() => this.GetEnumerator();
+        IEnumerator<T> IEnumerable<T>.GetEnumerator() =>
```

Comment:
nit: I prefer to use the expression-bodied methods for one-liners, and this isn't one-line any more.

Reply:
Changed